src/Test.FunctionalTests.BinSkim.Driver/BaselineTestsData/Expected/Binskim.linux-x64.dll.sarif

```diff
@@ -373,7 +373,7 @@
             "id": "NotApplicable_InvalidMetadata",
             "arguments": [
               "Binskim.linux-x64.dll",
-              "EnableAdditionalSdlSecurityChecks",
+              "EnableMicrosoftCompilerSdlSwitch",
```

Comment:
>"EnableMicrosoftCompilerSdlSwitch", [](http://example.com/codeflow?start=14&length=35)

The only change for all of these sarif results is for the rule name change.

Reply:
Many of these sarif file changes will become obsolete if #590 is completed first.